inotify.go

```diff
@@ -245,6 +245,15 @@ func (w *Watcher) readEvents() {
 
 			mask := uint32(raw.Mask)
 			nameLen := uint32(raw.Len)
+
+			if mask&unix.IN_Q_OVERFLOW != 0 {
+				select {
+				case w.Errors <- errors.New("inotify queue overflow"):
```

Comment:
Agreed that we should change this. There is no precedent in the code right now, which contains several other errors that are just strings. I would be willing to merge this as is and improve error handling as a followup pull request.

I do think that the error should report that it's the inotify queue that overflowed, as it isn't a queue that's part of fsnotify that had a problem.